### PR TITLE
cmd/clef: add importraw feature to clef

### PR DESCRIPTION
**Pull request by holiman**
This adds `importraw` as a subcommand to clef. 

Walking the happy-path:
```
[user@work clef]$ go run . --suppress-bootwarn importraw ./hexkey
INFO [10-28|14:03:08.718] Starting clef                            keystore=/home/user/.ethereum/keystore light-kdf=false
DEBUG[10-28|14:03:08.719] FS scan times                            list="61.22µs" set="72.06µs" diff="6.886µs"
## Password

Please enter a password for the imported account
> 
-----------------------
## Password

Please repeat the password you just entered
> 
-----------------------
DEBUG[10-28|14:03:15.040] FS scan times                            list="78.702µs" set="74.598µs" diff="6.765µs"
## Info 
Key imported:
  Address 0x9160DC9105f7De5dC5E7f3d97ef11DA47269BdA6
  Keystore file: /home/user/.ethereum/keystore/UTC--2022-10-28T12-03-13.976383602Z--9160dc9105f7de5dc5e7f3d97ef11da47269bda6

The key is now encrypted; losing the password will result in permanently losing 
access to the key and all associated funds!

Make sure to backup keystore and passwords in a safe location.

```

### CODE REVIEW

File: cmd/clef/main.go [hunk 1]
```diff
@@ -434,6 +456,79 @@ func newAccount(c *cli.Context) error {
...
+	)
+	if c.Args().Len() != 1 {
+		return errors.New("<keyfile> must be given as first argument.")
+	}
+	var hexkey string
+	if data, err := os.ReadFile(c.Args().First()); err != nil {
+		return err
+	} else {
+		hexkey = strings.TrimSpace(string(data))
+	}
```
Comment (fjl) [inline cmd/clef/main.go:480]: Please use `crypto.LoadECDSA` here.

**Review by fjl — CHANGES_REQUESTED** [review]

**Comment by holiman** [comment]
This PR now also adds `clef` CLI tests, covering the `importraw`, `list-accounts` and `list-wallets` commands. cc @jmcook1186

File: cmd/clef/run_test.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	if err != nil {
+		return nil
+	}
+	t.Cleanup(func() {
+		os.RemoveAll(ddir)
+	})
+	return runWithKeystore(t, ddir, args...)
+}
+
+// runClef spawns geth with the given command line args and adds keystore arg.
```
Comment (holiman) [inline cmd/clef/run_test.go:74]:
```suggestion
// runClef spawns clef with the given command line args and adds keystore arg.
```

File: cmd/clef/run_test.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+func TestMain(m *testing.M) {
+	// check if we have been reexec'd
+	if reexec.Init() {
+		return
+	}
+	os.Exit(m.Run())
+}
+
+// runClef spawns geth with the given command line args and adds keystore arg.
```
Comment (holiman) [inline cmd/clef/run_test.go:60]:
```suggestion
// runClef spawns clef with the given command line args and adds keystore arg.
```

File: cmd/clef/run_test.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+type testproc struct {
+	*cmdtest.TestCmd
+
+	// template variables for expect
+	Datadir   string
+	Etherbase string
+}
+
+func init() {
+	// Run the app if we've been exec'd as "geth-test" in runGeth.
```
Comment (holiman) [inline cmd/clef/run_test.go:42]:
```suggestion
```

File: cmd/clef/run_test.go [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+	if err != nil {
+		return nil
+	}
+	t.Cleanup(func() {
+		os.RemoveAll(ddir)
+	})
+	return runWithKeystore(t, ddir, args...)
+}
+
+// runClef spawns clef with the given command line args and adds keystore arg.
```
Comment (holiman) [inline cmd/clef/run_test.go:73]:
```suggestion
// runWithKeystore spawns clef with the given command line args and adds keystore arg.
```

File: cmd/clef/consolecmd_test.go [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+		// Secondly, do a listing, using the same datadir
+		clef = runWithKeystore(t, clef.Datadir, "--suppress-bootwarn", "--lightkdf", "list-accounts")
+		if out := string(clef.Output()); !strings.Contains(out, "0x9160DC9105f7De5dC5E7f3d97ef11DA47269BdA6 (keystore:") {
+			t.Logf("Output\n%v", out)
+			t.Error("Failure")
+		}
+	})
+}
+
+// TestListAccounts tests clef --list-wallets
```
Comment (holiman) [inline cmd/clef/consolecmd_test.go:89]:
```suggestion
// TestListWallets tests clef --list-wallets
```

**Review by fjl — APPROVED** [review]

**Pull request merged**